Formula/postgresql@12.rb

```diff
@@ -81,22 +81,101 @@ def post_install
     return if ENV["CI"]
 
     (var/"log").mkpath
-    (var/"postgres").mkpath
-    unless File.exist? "#{var}/postgres/PG_VERSION"
-      system "#{bin}/initdb", "--locale=C", "-E", "UTF-8", "#{var}/postgres"
+    versioned_data_dir.mkpath
+    unless versioned_pg_version_exists?
+      system "#{bin}/initdb", "--locale=C", "-E", "UTF-8", versioned_data_dir
     end
```

Comment:
The syntax CI job wants me to change these 3 lines into this:
```suggestion
    system "#{bin}/initdb", "--locale=C", "-E", "UTF-8", versioned_data_dir unless versioned_pg_version_exists?
```

But for me, that line is now more unreadable. So I've left it as-is.

Reply:
I'll need to be changed, sorry.

Reply:
Ok. Addressed in ae24ea6696.